pkg/model/crd.go

```diff
@@ -618,6 +618,32 @@ func (r *CRD) ListOpMatchFieldNames() []string {
 	return r.cfg.ListOpMatchFieldNames(r.Names.Original)
 }
 
+// GetAllRenames returns all the field renames observed in the generator.yaml.
+func (r *CRD) GetAllRenames() (map[string]string, error) {
+	resourceConfig, ok := r.cfg.Resources[r.Names.Original]
+	if !ok {
+		return nil, fmt.Errorf("resource not found")
+	}
+
+	renames := make(map[string]string)
+	opMap := r.sdkAPI.GetOperationMap(r.cfg)
+	createOps := (*opMap)[OpTypeCreate]
+
+	if resourceConfig.Renames != nil && resourceConfig.Renames.Operations != nil {
```

Comment:
nit: Invert condition and early return to decrease nesting

Reply:
Nice suggestion. Changing in next push